### DISCUSSION TITLE
Are there any providers that don't hardcode the subtitles?

### DISCUSSION THREAD

**Discussion by NevermindNilas** [sdaqo/anipy-cli#203] [Q&A]
Hi,

Basically as the titles suggests, is there any provider that doesn't hardcode / hardburn the subtitles into the video stream or is there an option to fully turn off subtitles.

Thanks.

**Comment by sdaqo** [comment]
Not currently, no. If you know one tell me, I can look into it.

**Reply by candrapersada** [reply]
https://hianime.to/
![image](https://github.com/user-attachments/assets/97aff69a-e91b-427e-aeba-60b279194fb2)

**Comment by sdaqo** [comment]
Now there are, closing.
